VRCFaceTracking/Params/Data/UnifiedTrackingMutator.cs

```diff
@@ -0,0 +1,168 @@
+using System;
+using System.Drawing.Drawing2D;
+using System.Text.Json.Serialization;
+using System.Threading;
+using System.Windows.Markup;
+using VRCFaceTracking.Params;
+
+namespace VRCFaceTracking
+{
+    /// <summary>
+    /// Container of all functions and structures retaining to mutating the incoming Expression Data to be usable for output parameters.
+    /// </summary>
+    public class UnifiedTrackingMutator
+    {
+        public struct Mutation
+        {
+            public string Name;
+            public float Ceil; // The maximum that the parameter reaches.
+            public float Floor; // the minimum that the parameter reaches.
+            //public float SigmoidMult; // How much should this parameter be affected by the sigmoid function. This makes the parameter act more like a toggle.
+            //public float LogitMult; // How much should this parameter be affected by the logit (inverse of sigmoid) function. This makes the parameter act more within the normalized range.
+            public float SmoothnessMult; // How much should this parameter be affected by the smoothing function.
```

Comment:
Currently the calibrator includes only a ceiling and floor calibration for each parameter, and I included some commented out ones for adjusting the profile of each parameter (one that makes it more toggle like and one that makes it more curve like). Basically the Mutator class was really intended on VRCFT itself allowing users to modify how expressions appear or feel like on a user level, though how this class can be modified by modules isn't yet entirely clear.